pkg/views/ide/start-workspace.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package ide
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/charmbracelet/huh"
+	"github.com/daytonaio/daytona/pkg/views"
+)
+
+func RunStartWorkspaceForm(workspaceName string) bool {
+	var yesFlag bool
```

Comment:
Can we change this from `var yesFlag bool` to `confirmCheck := true`

Also, please change this file's name to start_workspace.go

Reply:
done